curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5efc4f528d6a74d05e68af74.md

```diff
@@ -73,6 +71,12 @@ const labelElem = document.querySelector('label[for="loving"]');
 assert(labelElem.textContent.replace(/\s/g, '').match(/Loving/i));
 ```
 
+There should be a space between your checkbox and your new `label` element.
+
+```js
+assert.match(code, /<input id=('|")loving\1 type=('|")checkbox\2> <label for=('|")loving\3>Loving<\/label>/);
```

Comment:
I would change this regex to be more flexible in case users changed any of the seed code. e.g. maybe they add a valid space in the input attributes - or change the order of the attributes. The test wouldn't pass, and it would say "There should be a space between your checkbox and your new `label` element." - which they might have. Maybe something like this:

```suggestion
assert.match(code, /> <label\s+for\s*=\s*('|")loving/);
```

Reply:
Great idea. Thank you.

Reply:
I added an additional space token so that a little thing like two or three spaces between the label and the the input element wouldn't break the test. What do you think?